### PR TITLE
feat: wait 60s before issueing a registry-change job

### PR DESCRIPTION
**Pull request by janl**
Closes https://github.com/greenkeeperio/greenkeeper/issues/244

### CODE REVIEW

File: lib/follow.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ const cleanDoc = require('normalize-registry-metadata')
 
 const env = require('./env')
 const rollbar = require('./rollbar')
+const asyncTimeout = require('./util').asyncTimeout
```
Comment (Realtin) [inline lib/follow.js]: is this syntax the same as `const {asyncTimeout} = require('./util')`?

Comment (janl) [reply lib/follow.js]: yes, it’s the pre-destructuring variant. I can bring that up to modern standards :)

File: test/follow.js [hunk 2]
```diff
@@ -161,7 +162,13 @@ const {
         }
       }
     })
-    const job = await channel.get(env.QUEUE_NAME)
+
+    let job
+    do {
+      job = await channel.get(env.QUEUE_NAME)
+      await sleep(env.REGISTRY_CHANGE_DELAY / 2)
+    } while (job === false)
```
Comment (Realtin) [inline test/follow.js]: is that `while (job === false)` safe? what if the job cant be create because of something else?

Comment (janl) [reply test/follow.js]: we’ll get a 30s timeout from the test suite if this spins longer than we need it to.

But I can add a try/catch :)

**Pull request merged**